packages/upward-js/lib/resolvers/ComputedResolver.js

```diff
@@ -0,0 +1,18 @@
+const debug = require('debug')('upward-js:ComputedResolver');
+const AbstractResolver = require('./AbstractResolver');
+
+class ComputedResolver extends AbstractResolver {
+    static get resolverType() {
+        return 'computed';
+    }
+    static get telltale() {
+        return 'computed';
+    }
+    async resolve() {
+        debug('Computed resolver is meant for UPWARD PHP only.');
+
+        return '';
```

Comment:
Wouldn't we be able to at least supply webpageChunks based on the page type similar to how the php implementation globs for those files?

Reply:
@fooman Not without the page type information unfortunately. Each root component is its own chunk (sometimes more than one), and we only inline the chunks for the current page type.